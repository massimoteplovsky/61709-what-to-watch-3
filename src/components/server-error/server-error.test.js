import React from "react";
import renderer from "react-test-renderer";
import ServerError from "./server-error.jsx";

it(`<ServerError /> component renders correctly`, () => {
  const tree = renderer
    .create(<ServerError/>);
  expect(tree).toMatchSnapshot();
});
