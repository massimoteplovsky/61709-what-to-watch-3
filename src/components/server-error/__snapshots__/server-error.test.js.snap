// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ServerError /> component renders correctly 1`] = `
<div
  className="user-page"
>
  <h1>
    Sorry, connection with server failed!
  </h1>
</div>
`;
